frontend/src/components/RequestDialog.vue

```diff
@@ -122,26 +116,69 @@ const submitRequestTransaction = async (formResult: RequestFormResult) => {
     throw new Error('No signer available!');
   }
 
-  await runTransfer(
-    formResult,
-    provider.value,
-    signer.value,
-    requestManagerAddress.value,
-    requestFeeAmount.value,
-    configuration.chains,
+  const sourceChainConfiguration = configuration.chains[formResult.sourceChainId.value];
+  const sourceChain = {
```

Comment:
I would add an explicit type to this object (`Chain`).

Or maybe even better add a utility function for generating a `Chain` from a `ChainWithTokens` object. This seems to be a pretty common task.

Reply:
In regards of [this comment](https://github.com/beamer-bridge/beamer/pull/651#discussion_r874732102) I'll do it there.

Reply:
✔️